.github/workflows/visual-testing.yml

```diff
@@ -9,12 +9,15 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v3.0.0
+      - uses: pnpm/action-setup@v2.2.1
+        with:
+          version: 6.16.0
```

Comment:
Just saw that this is optional and is based on packageManager


https://github.com/pnpm/action-setup#version

Reply:
Good news!